crates/rspack_plugin_copy/src/lib.rs

```diff
@@ -526,7 +523,7 @@ impl CopyRspackPlugin {
             .lock()
             .expect("failed to obtain lock of `diagnostics`")
             .push(Diagnostic::error(
-              "CopyRspackPlugin Error".into(),
+              "CopyPlugin".into(),
```

Comment:
Both `Diagnostic::warn` and `Diagnostic::error` are legacy APIs. So the `title` is not the same as which in the `TraceableError`. I prefer to align this part with `TraceableError`.